src/main/webapp/js/instructorFeedbackEdit/questionRubric.es6

```diff
@@ -46,8 +46,8 @@ function addRubricRow(questionNum) {
         </tr>`;
 
     // Row to insert new row after
-    const lastRow = $(`#rubricEditTable-${questionNum} tr:last`);
-    $(newRubricRow).insertAfter(lastRow);
+    const secondLastRow = $(`#rubricEditTable-${questionNum} tbody tr:nth-last-child(2)`);
```

Comment:
While we're at it, maybe we should prefix this variable name with a $ ie `$secondLastRow`. This was a convention that was made to distinguish jQuery objects from other variables, though older code may not have been updated to follows this.